plantuml-maven-plugin/src/main/java/com/github/davidmoten/plantuml/plugins/GenerateMojo.java

```diff
@@ -167,4 +176,22 @@ private static String commaSeparate(List<String> list) {
         return list.stream().collect(Collectors.joining(","));
     }
 
+	private void extractPreproc(SourceFileReaderExt sourceFileReader) throws IOException {
+		for (BlockUml blockUml : sourceFileReader.getBlocks()) {
+			final SuggestedFile suggested = sourceFileReader.extractSuggestedFile(blockUml)
+					.withPreprocFormat();
+			final SFile file = suggested.getFile(0);
+			getLog().info("Export preprocessing source to " + file.getPrintablePath());
```

Comment:
`Export` -> `exporting` please (note lower case for logging consistency for this project)